elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java

```diff
@@ -492,24 +515,28 @@ private boolean validateTableConfig() {
         for (Table table : elideTableConfig.getTables()) {
 
             validateSql(table.getSql());
+            validateArguments(table.getArguments());
             Set<String> tableFields = new HashSet<>();
 
             table.getDimensions().forEach(dim -> {
                 validateFieldNameUniqueness(tableFields, dim.getName(), table.getName());
                 validateSql(dim.getDefinition());
                 validateTableSource(dim.getTableSource());
+                validateArguments(dim.getArguments());
                 extractChecksFromExpr(dim.getReadAccess(), extractedChecks, visitor);
             });
 
             table.getMeasures().forEach(measure -> {
                 validateFieldNameUniqueness(tableFields, measure.getName(), table.getName());
                 validateSql(measure.getDefinition());
+                validateArguments(measure.getArguments());
                 extractChecksFromExpr(measure.getReadAccess(), extractedChecks, visitor);
             });
 
             table.getJoins().forEach(join -> {
                 validateFieldNameUniqueness(tableFields, join.getName(), table.getName());
-                validateJoin(join);
+                validateSql(join.getDefinition());
```

Comment:
validate Join seems to have done more validation.  Is it safe to remove that extra validation?

Reply:
Existing validations for join's definition will not work with new syntax. I will add validations for Join's definition in another PR along with validation for Measure and Dimension definitions.